src/community/jdbcconfig/src/test/java/org/geoserver/jdbcconfig/internal/JDBCConfigPropertiesTest.java

```diff
@@ -17,6 +12,15 @@
 import org.junit.Before;
 import org.junit.Test;
 
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+import static org.geoserver.jdbcconfig.JDBCConfigTestSupport.createTempDir;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.junit.Assert.*;
```

Comment:
Is this just a change of imports? Wonder if your IDE is sorting things in a different order than eclipse?